tests/test_fips.py

```diff
@@ -320,3 +320,34 @@ def test_icainfo_binary(container_per_test: ContainerData) -> None:
     )
     container_per_test.connection.check_output("icastats -k")
     container_per_test.connection.check_output("icastats -S")
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_TESTER_IMAGES, indirect=True
+)
+def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
+    container_per_test.connection.check_output(
+        "zypper -n install mozilla-nss-tools"
+    )
+    # Seed file
+    container_per_test.connection.check_output(
+        "dd if=/dev/urandom bs=1k count=2048 of=seedfile.dat"
+    )
+    # Empty password
+    container_per_test.connection.check_output("touch password.txt")
+    # Create database, needed prerequisite, needs one <enter> as interaction
```

Comment:
The enter part of the comment is invalid, the rest is also of questionable usefulness, but not wrong
```suggestion
    # Create database, needed prerequisite
```

Reply:
Changed :heavy_check_mark: